packages/esm-ward-app/src/types/index.ts

```diff
@@ -176,10 +176,7 @@ export interface EncounterPayload {
 }
 
 export interface ObsPayload {
-  concept: Concept;
+  concept: Concept | string;
```

Comment:
String is for the case when we are sending back a uuid, correct?

Reply:
Yes